groovy-transform-app-dependencies/pom.xml

```diff
@@ -9,22 +9,32 @@
     <description>Spring Cloud Stream Groovy Transform App Dependencies</description>
 
     <parent>
-	<artifactId>spring-cloud-dependencies-parent</artifactId>
+    <artifactId>spring-cloud-dependencies-parent</artifactId>
```

Comment:
This is wrong again.
Please, be sure that you use tabs for indents and reformat the whole file if that

Reply:
it looks ok in my editor. not sure why its showing like here

Reply:
Right, because you indents are whitespaces meanwhile all other here in this file are tab.
Or otherwise.
We prefer tabs for code files. That's why I'm asking you to be sure that they are tabs and reformat this file for the solid code style.

Thanks